src/modules/adaptors/Frax/FTokenAdaptor.sol

```diff
@@ -29,6 +29,17 @@ contract FTokenAdaptor is BaseAdaptor {
      */
     error FTokenAdaptor__FTokenPositionsMustBeTracked(address fToken);
 
+    /**
+     * @notice This bool determines how this adaptor accounts for interest.
```

Comment:
For pairs following `FraxlendPairCorev1` are we suggesting strategists to have this set as false, and then optional for `FraxlendPairCorev2`?